src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -48,7 +48,7 @@ public IReadOnlyList<object> Attributes
         {
             get
             {
-                return _details.Attributes;
+                return _details.Attributes.Attributes;
```

Comment:
Moot point - `Attributes` is only defined on `DefaultModelMetadata`. So really this isn't exposed unless you downcast.

I can change this to expose `ModelAttributes` instead